### PR TITLE
[GLO-1886] move LineItem_AppKey to prevent GetSalesOrder SDK errors

### PR DESCRIPTION
**Pull request by gabrielsond**
AppKey is nested inside LineItem in the [proto](https://github.com/vendasta/vendastaapis/blob/834f43ffb4ce773147ecbac195aa6609ef8e332a/sales/v1/workflow.proto#L57). Due to this, the PHP SDK generation is not working right for calling GetSalesOrders

These changes modify the generated code so that AppKey works and calls can continue. This moves AppKey to a new class file, updates references to the new class, and tests.

I have testCreateAndActivateOrderAndGetSalesOrder working locally with valid VNDR demo values.

GLO-1886

@vendasta/lucky-charms

### CODE REVIEW

File: tests/SalesOrdersClientTest.php [hunk 1]
```diff
@@ -119,4 +120,49 @@ public function testCreateAndActivateOrderForAddon() {
...
+        }
+        self::assertNotEmpty(
+            $resp1->getOrderId(),
+            'expected order ID'
+        );
+
+        // Create second request to get first order
+        $req2 = new GetSalesOrderRequest();
+        $req2->setOrderId($resp1->getOrderId());
+        $req2->setBusinessId("AG-GPDTJKCPMD");
```
Comment (coreyhickson) [inline tests/SalesOrdersClientTest.php]:
```suggestion
        $req2->setBusinessId("AG-123");
```

**Review by coreyhickson — APPROVED** [review]
My only concern is I think we should solve this the same way in both SDKs, so if we're gonna go this way, we should update https://github.com/vendasta/g-suite-php-sdk/pull/3 so that it does the rename rather than move them into the same file

**Comment by gabrielsond** [comment]
> My only concern is I think we should solve this the same way in both SDKs, so if we're gonna go this way, we should update [vendasta/g-suite-php-sdk#3](https://github.com/vendasta/g-suite-php-sdk/pull/3) so that it does the rename rather than move them into the same file

I tried that method first but I couldn't get AppKey to work inside LineItem.

**Comment by coreyhickson** [comment]
Right but did we try this route in G Suite? My point being the SDKs should be the same, I don't want to have different manual fixes for different SDKs if possible

**Comment by gabrielsond** [comment]
> Right but did we try this route in G Suite? My point being the SDKs should be the same, I don't want to have different manual fixes for different SDKs if possible

Created https://github.com/vendasta/g-suite-php-sdk/pull/6 to deal with this issue.

**Pull request merged**